guides/process/scheduling/how_to_timesheet.md

```diff
@@ -40,6 +40,12 @@ You may be asked to support internal Made Tech initiatives while on a billed eng
 
 Where these commitments total less than 5 hours per week, they should be included in our standard operating overhead and should not be accounted for separately from billed hours.
 
+Note that some internal activities do not count towards this 5 hours as our clients are perceived to benefit from them indirectly. These include:
+
+* Hiring
+* Line management (1:1s etc.)
+* Taking part in communities of practice
```

Comment:
Any more we should add here?

Reply:
I think this is a bit confusing. Hiring *is* included for the purposes of the 5 hour break point, as explicitly stated in the first line of this section. Line management and CoP involvement (including CARE) are not.

Reply:
Ah, I obviously got the wrong end of the stick from our meeting. kind of proves the value of this exercise! Will remove hiring.